EKrut-Server/src/ekrut/server/managers/ServerSessionManager.java

```diff
@@ -83,15 +105,52 @@ public UserResponse logoutUser(String username) throws SQLException {
 	 *
 	 * @param username The username of the user.
 	 * @return `true` if the user is logged in, `false` if the user is not logged in or if a database error occurred.
-	 * @throws SQLException if a database error occurred.
 	 */
-	public boolean isLoggedin(String username) throws SQLException {
+	public boolean isLoggedin(String username) {

```

Comment:
It might be useful to know if a specific user is currently logged in but there should also be a method that checks if a `ConnectionToClient` has a logged in user associated with it.